### ISSUE TITLE
fonts fail to build with fontmake

### ISSUE THREAD

**Issue by fabiangreffrath** [madmalik/mononoki#99]
Hi,

today I tried to get the mononoki Debian package upgraded fom 1.2 to 1.5, however, without success:

```sh
$ fontmake -o ttf -g src/mononoki.glyphs 
INFO:fontmake.font_project:Building master UFOs and designspace from Glyphs source
INFO:glyphsLib.classes:Parsing "src/mononoki.glyphs" file into <GSFont>
fontmake: Error: In 'src/mononoki.glyphs': Loading Glyphs file failed: 'NoneType' object has no attribute 'groups'
```

And another error when using UFO instead of GLYPHS:

```sh
$ fontmake -o ttf -u src/mononoki-Regular.ufo
INFO:fontmake.font_project:Building TTF for mononoki-Regular
INFO:ufo2ft:Pre-processing glyphs
INFO:ufo2ft.filters.base:Running DecomposeComponentsFilter on mononoki-Regular
INFO:ufo2ft.filters.base:Running RemoveOverlapsFilter on mononoki-Regular
ERROR:ufo2ft.filters.removeOverlaps:Failed to remove overlaps for uni00B5
fontmake: Error: In 'src/mononoki-Regular.ufo': Compiling UFO failed: Unhandled open contour
```

Package versions:

```sh
fontmake          2.4.1-2
python3-glyphslib 6.0.6+ds1-1
python3-ufo2ft    2.28.0-1
```

**Comment by madmalik** [comment]
Looking into this. The error with the UFO file was apparently a problem with the contour of the µ glyph. I have redrawn that glyph and it fixed the error for me, I was able to create the font files with fontmake from the UFO files.

I've pushed the changes to main, could you test if it works for you with the new UFO files?

Using the GLYPHS file with fontmake still fails, but that looks like a bug in fontmake to me. The error is basically the python equivalent of a NPX, so nothing I can fix from my side.

**Comment by fabiangreffrath** [comment]
> I've pushed the changes to main, could you test if it works for you with the new UFO files?

Yes, now the font builds from the UFO file. Thanks!

> Using the GLYPHS file with fontmake still fails, but that looks like a bug in fontmake to me. The error is basically the python equivalent of a NPX, so nothing I can fix from my side.

I'll forward this to upstream, thanks for looking into this.

**Comment by felis-blue** [comment]
Fixed in version 1.6
